src/pages/Intro.js

```diff
@@ -2,12 +2,16 @@ import React from 'react';
 import { Link } from 'react-router-dom';
 
 import { Routes } from '@/constants';
+import Container from 'react-bootstrap/Container';
```

Comment:
NIT: I think we should follow a consistent import style where we import node modules first, then custom after

Reply:
updated. please verify

Reply:
Awesome, thanks